Userland/Utilities/top.cpp

```diff
@@ -70,6 +88,8 @@ struct Snapshot {
     u32 sum_times_scheduled { 0 };
 };
 
+static bool parse_args(int argc, char** argv, TopOption& top_option);
```

Comment:
This fwd-declaration is not particularly useful, just make the function definition include `static` instead?

Reply:
You're totally right about that, making it a static void instead as well. Thanks!